test/integration/ListPopFront/list popFront.gls

```diff
@@ -0,0 +1,3 @@
+-
```

Comment:
Code looks really good. Nice!

One thing: this should be named `list pop front.gls` to go along with the other test files.